doc/source/data/working-with-llms.rst

```diff
@@ -19,10 +19,18 @@ Perform batch inference with LLMs
 At a high level, the `ray.data.llm` module provides a `Processor` object which encapsulates
 logic for performing batch inference with LLMs on a Ray Data dataset.
 
-You can use the `build_llm_processor` API to construct a processor. In the following example, we use the `vLLMEngineProcessorConfig` to construct a processor for the `meta-llama/Llama-3.1-8B-Instruct` model.
+You can use the `build_llm_processor` API to construct a processor.
+In the following example, we use the `vLLMEngineProcessorConfig` to construct a processor for the `meta-llama/Llama-3.1-8B-Instruct` model.
+
+To run this example, install vLLM, which is a popular and optimized LLM inference engine.
+
+.. testcode::
+
+    pip install -U vllm
```

Comment:
Better to pin the version. You can add one sentence saying later versions should still work but not guaranteed.